### PR TITLE
Fix/lastminutefixes

### PR DESCRIPTION
**Pull request by GoOz**
Last minute fixes. 

Some issues after the last refactoring were well hidden.

### CODE REVIEW

File: stylus/components/popover.styl [hunk 1]
```diff
@@ -30,7 +30,7 @@ $popover
     button
     [role=button]
         display          block
-        padding          em(8px) em(32px) em(8px) em(42px)
+        padding          em(8px, 16px) em(32px, 16px) em(8px, 16px) em(42px, 16px)
```
Comment (ptbrowne) [inline stylus/components/popover.styl]: I wonder if we could use rem here ?

File: react/Button/styles.styl [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+.coz-btn
+    @extends .c-btn
+
+.coz-btn--regular
+    @extends .c-btn--regular
+
+.coz-btn--secondary
+
+    @extends .c-btn--secondary
```
Comment (ptbrowne) [inline react/Button/styles.styl]: extra whitespace ?

Comment (GoOz) [reply react/Button/styles.styl]: Good catch! Thanks.

File: stylus/components/popover.styl (hunk 1 above)
Comment (GoOz) [reply stylus/components/popover.styl]: Indeed, that would in fact be a way better solution. I forgot about this, thanks.

**Comment by ptbrowne** [comment]
Thx !

**Pull request merged**